common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -22,8 +22,39 @@
 // THE SOFTWARE.
 package com.microsoft.identity.common.java.util;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+
+import lombok.NonNull;
+
 public class StringUtil {
+    /**
+     * The constant ENCODING_UTF8.
+     */
+    public static final String ENCODING_UTF8 = "UTF-8";
+
+    /**
+     * checks if string is null or empty.
+     *
+     * @param message String to check for null or blank.
+     * @return true, if the string is null or blank.
+     */
     public static boolean isNullOrEmpty(String message) {
         return message == null || message.trim().length() == 0;
     }
```

Comment:
We can probably get rid of this method and just use `StringUtils.isNullOrEmpty`

Reply:
I can't find that method though.

Reply:
Ohh sorry, this is the that method 😂

Reply:
At some point, we should write this to not construct a new string in order to determine if it only contains whitespace.